docs/content/how-to/aws/etc-backup-restore.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Etcd backup/restore (manual process)
+---
+
```

Comment:
note aside: as we move towards beta API we need to consider introducing feature gated functionality